package.json

```diff
@@ -87,6 +85,7 @@
         "update:spectrum-css:cleanup": "yarn --ignore-scripts && yarn process-spectrum && yarn lint",
         "update:spectrum-css:nonbreaking": "node ./scripts/update-spectrum-css.js || yarn update:spectrum-css:cleanup",
         "verify-build-artifacts": "node ./scripts/verify-build-artifacts.js",
+        "version:update": "genversion --verbose --semi --esm ./tools/base/src/version.js",
```

Comment:
This is the same as what was in the update-versions.js node script but leveraging the command line tool instead.